packages/core-web/src/index.js

```diff
@@ -134,11 +134,15 @@ function removeTemporaryStyles() {
   jQuery('.temp-dropdown-placeholder').remove();
 }
 
-function executeAfterCreatedRoutines() {
-  removeTemporaryStyles();
+// Changes 'style-bypass-vue-compilation' tags back into 'style' tags
+function restoreStyleTags() {
+  jQuery('style-bypass-vue-compilation')
+    .replaceWith(`<style>${jQuery('style-bypass-vue-compilation').html()}</style>`);
```

Comment:
hmm this may not work if the `<style>` tag contains conflicting html syntax

Reply:
As per our discussion, the problem here is that `<style>` tag has a special parsing treatment, akin to `<script>` tag. Thus, if I simply change `<style>` to `<style-bypass-vue-compilation>` the block of code within the tag will be parsed as a normal html and there's a chance there may be invalid html syntax. This will affect the parsing of the page when I compile it with `vue-template-compiler`. 

To workaround it, I've piggybacked on the idea we used for `<script>` tags (`type="application/javascript"`) so that we can get the Vue compiler to recognize it as a script and ignore it during compilation (without discarding it). To make sure that it does not get executed as a script, I've added `src` and `defer` attributes to delay the execution of the script (as it is not meant to be an actual script, it was simply transformed into script tag to bypass Vue compilation), until our MarkBind plugin can restore it back into a proper `<style>` tag. 

I believe the above workaround should resolve the problem you mentioned. I've played around with it and so far it seems to be working fine (browser treatment of this script tag seems to be alright). 

Another alternative would be to transform the `<style>` tag to `<textarea>` tag. It will give us the same behaviour as above but the html syntax contained within the tag will be escaped instead.